onboard-app-guilherme/app/users.tsx

```diff
@@ -1,9 +1,39 @@
-import { Text, View } from "react-native";
+import { useQuery } from "@apollo/client/react";
+import { ActivityIndicator, FlatList, Text, View } from "react-native";
+import { UserItem } from "../src/components/user-item";
+import { USERS_QUERY } from "../src/graphql/queries";
+import type { UsersQueryResponse } from "../src/graphql/types";
 
 export default function UsersScreen() {
+  const { data, loading, error } = useQuery<UsersQueryResponse>(USERS_QUERY);
```

Comment:
Aqui é bom dar um skip quando o usuário não está logado também, pra evitar que a query seja feita só pra retornar um erro
```suggestion
  const { data, loading, error } = useQuery<UsersQueryResponse>(USERS_QUERY, { skip: !isLogged });
```